src/radical/pilot/agent/rm/ccm.py

```diff
@@ -54,7 +54,12 @@ def _configure(self):
         # Some simple arithmetic
         self.cores_per_node = ccm_nodes_length / ccm_node_list_length
         self.gpus_per_node  = self._cfg.get('gpus_per_node', 0) # FIXME GPU
-        self.lfs_per_node   = {'path' : self._cfg.get('lfs_path_per_node', None),
+
+        lfs_path = self._cfg.get('lfs_path_per_node', None)
+        if '$' in path:
+            lfs_path = os.path.expandvars(path)
```

Comment:
The dict getter defaults to `None`, but:
```
>>> os.path.expandvars(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/posixpath.py", line 287, in expandvars
    if '$' not in path:
TypeError: argument of type 'NoneType' is not iterable
```

I guess this should  be either one  of these:

```
lfs_path = self._cfg.get('lfs_path_per_node', None)
if lsf_path: 
    lfs_path = os.path.expandvars(lsf_path)
```

or:

```
lfs_path = self._cfg.get('lfs_path_per_node', '')
lfs_path = os.path.expandvars(lsf_path)
```

I like the second a little more, but either one  works.

Reply:
Okay! Fixing!